storagenode/storagenodedb/storageusage.go

```diff
@@ -123,22 +119,49 @@ func (db *storageusageDB) GetDailyTotal(ctx context.Context, from, to time.Time)
 	var stamps []storageusage.Stamp
 	for rows.Next() {
 		var atRestTotal float64
-		var timeStamp time.Time
+		var intervalStart time.Time
 
-		err = rows.Scan(&atRestTotal, &timeStamp)
+		err = rows.Scan(&atRestTotal, &intervalStart)
 		if err != nil {
 			return nil, err
 		}
 
 		stamps = append(stamps, storageusage.Stamp{
-			AtRestTotal: atRestTotal,
-			Timestamp:   timeStamp,
+			AtRestTotal:   atRestTotal,
+			IntervalStart: intervalStart,
 		})
 	}
 
 	return stamps, nil
 }
 
+// Summary returns aggregated storage usage across all satellites
+func (db *storageusageDB) Summary(ctx context.Context, from, to time.Time) (_ float64, err error) {
+	defer mon.Task()(&ctx)(&err)
+	var summary sql.NullFloat64
+
+	query := `SELECT SUM(at_rest_total) 
+				FROM storage_usage
+				WHERE ? <= interval_start AND interval_start <= ?`
+
+	err = db.QueryRowContext(ctx, query, from.UTC(), to.UTC()).Scan(&summary)
+	return summary.Float64, nil
+}
+
+// SatelliteSummary returns aggregated storage usage for particular satellite
+func (db *storageusageDB) SatelliteSummary(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) (_ float64, err error) {
+	defer mon.Task()(&ctx)(&err)
+	var summary sql.NullFloat64
```

Comment:
cannot it just be `float64`?

According the query it doesn't seem that it can be nil.

Reply:
`SUM(at_rest_total)` returns `NULL` when the table is empty.

Reply:
Thanks, that's what I was missing, I thought that when no rows it will return nothing (i.e. `ErrNoRows`)